docs/guides/src/main/server/caching.adoc

```diff
@@ -10,52 +10,110 @@ includedOptions="cache cache-*">
 Keycloak is designed for high availability and multi-node clustered setups.
 The current distributed cache implementation is built on top of https://infinispan.org[Infinispan], a high-performance, distributable in-memory data grid.
 
-All available cache options are build options, so they need to be applied to a `build` of Keycloak before starting.
-
-== Enable distributed caching
+== Enable clustering
```

Comment:
afair we decided to always use cache instead of cluster, but not hard on this.

Reply:
Glad you are not hard on it. distributed caching seems more verbose and the same thing as clustering.

But I'll change it back to avoid more discussions.